extensions/big-data-cluster/src/bigDataCluster/dialog/bdcDashboardOverviewPage.ts

```diff
@@ -244,11 +244,11 @@ function createServiceEndpointRow(modelBuilder: azdata.ModelBuilder, container:
 		const endpointCell = modelBuilder.hyperlink()
 			.withProperties({ label: endpoint.endpoint, url: endpoint.endpoint, CSSStyles: { 'height': '15px' } })
 			.component();
-		endPointRow.addItem(endpointCell, { CSSStyles: { 'width': `${serviceEndpointRowEndpointCellWidth}px`, 'min-width': `${serviceEndpointRowEndpointCellWidth}px`, 'overflow': 'hidden', ...cssStyles.hyperlink } });
+		endPointRow.addItem(endpointCell, { CSSStyles: { 'width': `${serviceEndpointRowEndpointCellWidth}px`, 'min-width': `${serviceEndpointRowEndpointCellWidth}px`, 'overflow': 'hidden', 'text-overflow': 'ellipsis', ...cssStyles.hyperlink } });
```

Comment:
something to think about: can we move these properties into .css files?

Reply:
Yeah that's on my backlog of features to add - since as far as I'm aware there isn't a way for an extension to contribute style sheets at this time.